pkg/controller/controller.go

```diff
@@ -637,29 +637,81 @@ func (c *controller) getClusterServiceBrokerForServiceBinding(instance *v1beta1.
 	return broker, nil
 }
 
-func (c *controller) getBrokerClientForServiceBinding(instance *v1beta1.ServiceInstance, binding *v1beta1.ServiceBinding, broker *v1beta1.ClusterServiceBroker) (osb.Client, error) {
-	pcb := pretty.NewInstanceContextBuilder(instance)
-	authConfig, err := getAuthCredentialsFromClusterServiceBroker(c.kubeClient, broker)
-	if err != nil {
-		s := fmt.Sprintf("Error getting broker auth credentials for broker %q: %s", broker.Name, err)
-		glog.Warning(pcb.Message(s))
-		c.updateServiceBindingCondition(
-			binding,
-			v1beta1.ServiceBindingConditionReady,
-			v1beta1.ConditionFalse,
-			errorAuthCredentialsReason,
-			"Error getting auth credentials. "+s,
-		)
-		c.recorder.Event(binding, corev1.EventTypeWarning, errorAuthCredentialsReason, s)
-		return nil, err
-	}
+func (c *controller) getBrokerClientForServiceBinding(instance *v1beta1.ServiceInstance, binding *v1beta1.ServiceBinding) (osb.Client, error) {
 
-	clientConfig := NewClientConfigurationForBroker(broker.ObjectMeta, &broker.Spec.CommonServiceBrokerSpec, authConfig)
+	var brokerClient osb.Client
 
-	glog.V(4).Infof("Creating client for ClusterServiceBroker %v, URL: %v", broker.Name, broker.Spec.URL)
-	brokerClient, err := c.brokerClientCreateFunc(clientConfig)
-	if err != nil {
-		return nil, err
+	if instance.Spec.ClusterServiceClassSpecified() {
+
```

Comment:
nit: what is with all this extra space in the new block?

Reply:
I can fix it if you want, it made it easier for me to read and see the if blocks especially when I was moving things around. But now that it's done I'd be happy to remove it if required.

Reply:
I did it in controller_binding.go as well. Let me know if you really want me to fix this nit or not.